sos/plugins/ipa.py

```diff
@@ -120,14 +120,24 @@ def setup(self):
         self.add_forbidden_path("/etc/dirsrv/slapd-*/pwdfile.txt")
         self.add_forbidden_path("/etc/named.keytab")
 
+        ### Make sure to lookup the right NSS DB for PKI certs
+        if ipa_version == "v3":
+            self.add_cmd_output([
+                "certutil -L -d %s/alias" % self.pki_tomcat_dir_v3
+            ])
```

Comment:
No need to pass a list for a single command: just `self.add_cmd_output("command string")`.